### PR TITLE
new speaker : xavier blanc

### PR DESCRIPTION
**Pull request by xblanc33**
Dear All,

Here is my modest contribution.

I hope the PR is ok.

Regards

X.B.

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -14712,6 +14711,69 @@ var data = {
...
+    {
+      since: '2018-06-09',
+      name: 'Xavier Blanc',
+      bio:
+        "Développeur (et accessoirement Professeur à l'Université de Bordeaux), toujours curieux des nouvelles approches de programmation (Clean Code, Craftmanship, DDD, BDD, Dette Technique, etc.).",
+      picture: 'https://avatars3.githubusercontent.com/u/10332041?s=460&v=4',
+      cities: ['Bordeaux', 'Paris'],
+      contacts: {
+        mail: 'xavier.blanc@u-bordeaux.fr',
+        websites: [
```
Comment (dadoonet) [inline baggers.js]: Websites is not part of contacts. Please remove that part.

File: baggers.js [hunk 2]
```diff
@@ -14712,6 +14711,69 @@ var data = {
...
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/xavier-blanc-3b9785a/',
+        },
+        {
+          name: 'GitHub',
+          url: 'https://github.com/xblanc33',
+        },
+        {
+          name: 'YouTube',
```
Comment (dadoonet) [inline baggers.js]: Although YouTube is not part of the official list, I'm +1 to add it. (http://data.brownbaglunch.fr/#websites)

**Review by dadoonet — CHANGES_REQUESTED** [review]

File: baggers.js [hunk 3]
```diff
@@ -14712,6 +14711,59 @@ var data = {
...
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/xavier-blanc-3b9785a/',
+        },
+        {
+          name: 'GitHub',
+          url: 'https://github.com/xblanc33',
+        },
+        {
+          name: 'Web',
```
Comment (dadoonet) [inline baggers.js:14738]: I was unclear but YouTube was fine. We just need to change the README as well then the website to add the YouTube icon IIRC.

**Review by dadoonet — APPROVED** [review]
LMK if you want me to merge it as is.

**Comment by xblanc33** [comment]
Should I do something ?

**Comment by dadoonet** [comment]
> Should I do something ?

Oh sorry if I was unclear but I was waiting for an answer on:

> LMK if you want me to merge it as is.

Unfortunately, we merged some other talks in the meantime so whatever your answer is (I merge or you revert last change), you'll have to rebase onto master first to resolve the current conflict.
LMK when it's done and I'll be happy to merge it. :)

**Pull request closed without merging**